core/Makefile

```diff
@@ -6,11 +6,13 @@ android:
 	cargo ndk --target x86_64-linux-android --android-platform $(MIN_VER) -- build --release
 
 	@echo "Cleaning up old .so's if they exist"
-	@rm -rf $(jniLibs)
+	@rm -rf $(jniLibs)/arm64-v8a
+	@rm -rf $(jniLibs)/armebi-v7a
+	@rm -rf $(jniLibs)/x86
+	@rm -rf $(jniLibs)/x86_64
 
 	@echo "Moving new .so's over"
-	@mkdir $(jniLibs)
-	@mkdir $(jniLibs)/arm64-v8a
+	@mkdir -p $(jniLibs)/arm64-v8a
```

Comment:
explain this change?

Reply:
The -p means it will create the sub directories of the folder you are trying to create if they don't already exist. So I can replace those two commands with just that.

Reply:
`-p` is good for things like this